src/Tutor.Core/InstructorModel/IInstructorService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using FluentResults;
+using Tutor.Core.DomainModel.KnowledgeComponents;
+using Tutor.Core.LearnerModel.DomainOverlay.EnrollmentModel;
+
+namespace Tutor.Core.InstructorModel;
+
+public interface IInstructorService
+{
+    Result<List<Course>> GetCourses(int userId);
```

Comment:
As with the learnerId, I imagine we can store the instructorId in the JWT? Maybe free the instructorService (an interface in the domain from the technological concept of userId?